emsdk.py

```diff
@@ -1452,7 +1452,7 @@ def emscripten_npm_install(tool, directory):
     closure_compiler_native = 'google-closure-compiler-linux'
   if MACOS and ARCH in ('x86', 'x86_64'):
     closure_compiler_native = 'google-closure-compiler-osx'
-  if WINDOWS and ARCH in ('x86_64', 'aarch64'):
+  if WINDOWS and ARCH in ('x86_64'):
```

Comment:
Ups, I didn't noticed that the original upstream code was:

```
if WINDOWS and ARCH == 'x86_64':
```

but nevermind.

Reply:
good you noticed that